packages/shared/lib/services/nango-config.service.ts

```diff
@@ -257,6 +257,18 @@ const parseModelInEndpoint = (endpoint: string, allModelNames: string[], inputMo
     return { success: true, error: null, response: inputModel };
 };
 
+const getEnabledStatus = (script: NangoIntegrationDataV2, isPublic: boolean, preBuilt: boolean): boolean => {
```

Comment:
what about calling this function `isEnabled`?

Reply:
Done with https://github.com/NangoHQ/nango/pull/1993/commits/d3a38c96d98ec76ea5b024a9a687418896f66451